foreman/client.py

```diff
@@ -116,7 +116,8 @@ class ForemanVersionException(Exception):
 
 class MethodAPIDescription(object):
     exclude_html_reg = re.compile('</?[^>/]+/?>')
-    resource_pattern = re.compile(r'^/api(/v[12])?/(?P<resource>\w+).*')
+    resource_pattern = re.compile(
+        r'^/(\bapi\b|\bkatello/api\b)(/v[12])?/(?P<resource>\w+).*')
```

Comment:
The `\b` here is not needed right? as we are already making sure that the char before and after is / in any case no? (Just making sure I'm not missing something).